rgw/v2/tests/s3_swift/test_bucket_lifecycle_object_expiration.py

```diff
@@ -95,13 +95,27 @@ def test_exec(config, ssh_con):
             value=str(config.rgw_lc_max_worker),
             ssh_con=ssh_con,
         )
-    log.info("trying to restart services")
-    srv_restarted = rgw_service.restart(ssh_con)
-    time.sleep(30)
-    if srv_restarted is False:
-        raise TestExecError("RGW service restart failed")
+
+    if config.test_lc_transition:
+        log.info("Set the Bucket LC transitions pre-requisites.")
+        reusable.prepare_for_bucket_lc_transition(config)
+        log.info(
+            "restart rgws after setting the storage placement configs for LC transition."
+        )
+        srv_restarted = rgw_service.restart(ssh_con)
```

Comment:
line no 105-110 is same as that of line no 113-118, so we can have it outside if-else condition